tests/test_selector.py

```diff
@@ -1132,3 +1132,69 @@ def test_set(self) -> None:
             ).extract(),
             ["url", "name", "startDate", "location", "offers"],
         )
+
+
+class SelectorTextTestCase(unittest.TestCase):
+
+    sscls = Selector
+
+    html_body = """
+            <body>
+                <div class="product">
+                    <div class="name">Product1</div>
+                    <span class="price"><b>Price:</b>100</span>
+                </div>
+                <div class="product">
+                    <div class="name">Product2</div>
+                    <span class="price"><b>Price:</b>200</span>
+                </div>
+            </body>
+            """
+
+    def test_text_get(self):
+        sel = self.sscls(text="<p>title:</h1>some text</h1></p>")
```

Comment:
```suggestion
        sel = self.sscls(text="<p>title:<h1>some text</h1></p>")
```

I wonder if this will alter the output.

Reply:
Was a typo in the string